ccloud-cube-demo/README.adoc

```diff
@@ -0,0 +1,241 @@
+= Confluent Cloud "The Cube" Demo
+
+Ricardo Ferreira, <ricardo@confluent.io>
+v1.00, January 4, 2019
+
+== Introduction
+
+This code contains a sample application called "The Cube" that allow users to interact with Kafka clusters running on Confluent Cloud by using their movile devices.
+
+While users play with the application from their mobile devices, events containing their moves are emitted to a topic. This topic holds the log of all events and it is the basis for a set of inbound streams created by KSQL. The game is about having people moving the cube in their mobile devices, trying to position the number 3 using a set of coordinates.
+
+image:images/the_cube.jpg[The Cube, 300, 500, role="center"]
+
+Anyone capable of perfectly positioning the cube to these coordinates will be shown in a KSQL query. This query is executed against a table called *SELECTED_WINNERS*, that is the result of a join between the inbound streams and a table that holds the information of the coordinates.
+
+image:images/selected_winner.png[Selected Winner]
+
+== Step 1: Create Kafka Cluster in Confluent Cloud
+
+1. Login to Confluent Cloud at https://confluent.cloud.
+
+2. Click *Create cluster*.
+
```

Comment:
Use `+` to denote subsequent line as part of the bullet point. i.e.: 

```
1. From the *Management -> Clusters* page, click the ellipses on the right-hand side of your cluster name and click *Client config*.

image:images/cloud-view-details.png[View Cluster Details]

2. Follow the on-screen Confluent Cloud installation instructions, starting from the section *Configure Confluent Cloud CLI*. You don’t need to perform any steps from that section, because you’re not installing the Confluent Cloud CLI. Just click *Create Kafka Cluster API key & secret* to generate a new credential key pair.

image:images/cloud-cli-config.png[Client Config]
```

Should be: 

```
1. From the *Management -> Clusters* page, click the ellipses on the right-hand side of your cluster name and click *Client config*.
+
image:images/cloud-view-details.png[View Cluster Details]

2. Follow the on-screen Confluent Cloud installation instructions, starting from the section *Configure Confluent Cloud CLI*. You don’t need to perform any steps from that section, because you’re not installing the Confluent Cloud CLI. Just click *Create Kafka Cluster API key & secret* to generate a new credential key pair.
+
image:images/cloud-cli-config.png[Client Config]
```

This ensures that ordered lists work. At the moment there are a lot of `1.` points :)